TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/build.gradle

```diff
@@ -0,0 +1,78 @@
+plugins {
+    id 'base'
+    id 'de.undercouch.download' version '5.5.0'
+}
+
+def apiPort = 8099 // Not a commonly used
+def openapiUrl = "http://127.0.0.1:${apiPort}/openapi.json"
+def serverPidFile = "${buildDir}/fastapi.pid"
+def openapiOutFile = "${buildDir}/openapi.json"
```

Comment:
nit: Are we able to use `layout.buildDirectory.file("fastapi.pid")` since buildDir is deprecated